README.md

```diff
@@ -502,3 +503,69 @@ Small to medium (depends on the choice of implemented lints or schema extensions
+
+Internationalizing Rust is a big, gradual project with many facets:
+- A lot of work is currently being done on internationalizing the compiler output ([tracking issue](https://github.com/rust-lang/rust/issues/100717), [dev docs](https://rustc-dev-guide.rust-lang.org/diagnostics/translation.html)). This is important work and both orthogonal to as well as a prerequisite for this project to work well.
+- There is [rouille](https://github.com/bnjbvr/rouille), which is both a joke and a starting point: It provides a procedural macro that translates all French keywords and function names to English before compilation. This means that error messages from the compiler will still have English function names.
+
+This project could go as follows:
+1. Implement one internationalization from the [tracking issue](https://github.com/rust-lang/rust/issues/100717) (if still available) to get accustomed to the compiler and tooling.
+2. Add a compiler plugin/patch so that it understands that tokens have a "canonical" and "local" description.
+3. Add a `Cargo.toml` entry for the canonical language and cargo commands for:
+   - Setting up an internationalized crate (source code, not output) This:
+     - `rustup install`s fluent.rs
+     - Creates relevant folders/files.
+     - Fills said files with all function names in the canonical language
+     - Adds a pre-commit hook for switching back to the canonical language.
+     - (stretch) checks all upstream crates for available translations.
+     - (further stretches) allows for translating used function names/structs from upstream crates and automatically creates a pull request, adding the translations to the upstream crate. Alternatively, integrate with pontoon.
+   - Switching locales: This translates *the entire crate* to another language and emits warnings for all names that are not translated.
+4. Implement the developed internationalization tooling in a fork of [rustlings](https://github.com/rust-lang/rustlings) as a testing ground.
+
+**Expected result**
+
+Rustlings can be -and maybe is- partially translated to another language.
+
+**Desirable skills**
+
+Intermediate-good knowledge of Rust. Especially being able to read and understand Rust source code (well-documented). Being a non-native English speaker probably helps with motivation. In any case, there will be a second language to translate to ;)
+
+**Project size**
+
+Medium to large (depending on how far the stretches are taken).
+
+**Difficulty**
+
+Medium to hard.
+
+**Mentor**
+- Fee (Fairy) Fladder ([GitHub](https://github.com/feefladder/), [Zulip](HELP))
```

Comment:
Isn't the mentor someone who's really familiar with the project?
I'm not sure whether you could mentor compiler or cargo changes :)